docs/inputs/table-group.md

```diff
@@ -17,26 +17,92 @@ Similar to an [**Input Group**](input-group), a **Table Group** allows the _**de
+For example, the following **Table Group** has three columns and 1 row. The header can be made using RichText input type and does NOT count as a row. The array name available for the **Code** step is `table`.
+
+<div style={{textAlign: 'center'}}>
+
+![img alt](/docs/inputs/table_group_numbered.png)
+
+</div>
+
+1. Drag the **Table Group** to arrange it vertically
+2. Specify a _**Title**_ that will be visible to the users
+3. Edit the **Table Group** properties (described below)
+4. Delete the **Table Group**
+5. Duplicate the last row in the current **Table Group** (similar to duplicate input but for the whole row)
+
+### Editing a table group
+
+To edit the table group, click the pencil icon near the top right side of the table group. A popup will appear with the following options:
 
 <div style={{textAlign: 'center'}}>
 
-![img alt](/docs/inputs/table_group.png)
+![img alt](/docs/inputs/edit_table_group_dialog.png)
 
 </div>
 
+1. **Name** -> this is the variable name of the table which will be used in the python code
+2. **Columns** -> the number of columns in the table
+
+| :trophy: The following are [premium](https://mecsimcalc.com/premium) features |
+| ----------------------------------------------------------------------------- |
+
+3. **Dynamic Rows** -> check this box if you want the user to be able to add and delete rows from the table  
+    After checking this box a list of options will appear:  
+   a) **_Max Rows_** -> The maximum number of rows that can be in the table (**NOT including the header row**)  
+   b) **_Min Rows_** -> The minimum number of rows that can be in the table
+   (**NOT including the header row**)  
+   c) **_Row Group Size_** -> The number of rows that are 'combined' together when the user 'Adds' or 'Deletes' rows. (_This should almost always be set to 1_)
+4. **Conditional Input Group** -> Similar to the [Conditional Input](https://docs.mecsimcalc.com//inputs/conditional-input) feature but for the entire **Input Group / Table Group**
+
 :::caution
-Don't forget to click on **APPLY** after changing the number of _columns_ or the array _name_ of the **Table Group**
+Don't forget to click on **APPLY** after editing the properties of the **Table Group**
+:::
+
+### Dynamic Rows
+
+The following embedded app demonstrates the Dynamic Rows feature using the **Table Group** properties shown above. Enter the values and press submit to calculate the weighted GPA. Try adding and deleting rows to see the output change!
+
+<div style={{width: "100%", height: "900px", overflow: "hidden"}}><iframe src="https://mecsimcalc.com/app/7646727/weighted_gpa_calculator_mobile_friendly" width="100%" height="100%" title="MecSimCalc" style={{position:"relative", left:"-45px", top:"-48px"}} frameBorder="0"></iframe></div>
```

Comment:
Can we see more of how you made the inputs? Maybe like images of the edit table properties and what the input editor looks like?